client/lib/plans/constants.js

```diff
@@ -346,11 +350,13 @@ export const PLANS_LIST = {
 			FEATURE_MANAGE
 		],
 		getSignupFeatures: () => [
-			FEATURE_STANDARD_SECURITY_TOOLS,
+			FEATURE_FREE_WORDPRESS_THEMES,
 			FEATURE_SITE_STATS,
+			FEATURE_STANDARD_SECURITY_TOOLS,
 			FEATURE_TRAFFIC_TOOLS
 		],
 		getBillingTimeFrame: () => i18n.translate( 'for life' ),
+		getSignupBillingTimeFrame: () => 'for life', //PLANS A/B TEST: Translate if test passes
```

Comment:
Any good reason to not wrap this one in a `translate` call immediately? This string is already translated in the above line.